### ISSUE TITLE
html-webpack-plugin-before-html-processing

### ISSUE THREAD

**Issue by newstackdevelopment** [stephenwf/create-nwb-webpack-config#3]
Not sure if it is here but it is in the trace.

``
Plugin could not be registered at 'html-webpack-plugin-before-html-processing'. Hook was not found.
BREAKING CHANGE: There need to exist a hook at 'this.hooks'. To create a compatibility layer for this hook, hook into 'this._pluginCompat'.
(node:9952) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
    at CaseSensitivePathsPlugin.apply (F:\Projects\Create_React_Apps\app-wrapper\node_modules\create-nwb-webpack-config\node_modules\case-sensitive-paths-webpack-plugin\index.js:136:12)
``

trying to use styleguidist 
```
var config = require("create-nwb-webpack-config")();

config.plugins = config.plugins.filter(function(plugin) {
  return !plugin.chunkNames && !(plugin.options && plugin.options.template);
});

process.traceDeprecation = true;

module.exports = {
  webpackConfig: config,
  components: "src/components/**/[A-Z]*.js"
};
```

**Comment by newstackdevelopment** [comment]
Ok saw that the problem is new version has no been published to npm.

**Comment by stephenwf** [comment]
ah that's my fault, I've published 1.0.1 now
